test/platform/events/streaming.spec.ts

```diff
@@ -0,0 +1,101 @@
+import expect from 'expect'
+import {
+  StreamingKeyResetEvent,
+  StreamingKeyRevokeEvent,
+  StreamingKeyExpiredEvent,
+  StreamingTimeExceededEvent,
+  StreamingPlaceUpdatedEvent,
+  Events
+} from '../../../src'
+
+describe('Streaming Events tests', () => {
+  it('StreamingKeyResetEvent static tests must pass', () => {
```

Comment:
As the tests exercise almost the same thing, with the `subType` being the only thing different, what do you think about using either a `describe.each` or a `test.each` to do these tests?